extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -154,6 +154,23 @@ export class ModelingStore extends DisposableObject {
     return this.state.get(this.activeDb);
   }
 
+  /**
+   * Returns the methods for the given database item and method signatures.
+   * If no method signatures are provided, returns all methods.
```

Comment:
This should probably be more specific. "no method signatures" can also be interpreted as giving an empty array, but this will actually return no methods because an empty array is truthy. I think that's fine, but we should just say that either the `methodSignatures` argument is not provided or the `methodSignatures` argument is `undefined`.

Reply:
Good point. I've clarified this comment.